dist/src/main/resources/application-tasklist.properties

```diff
@@ -34,7 +26,6 @@ camunda.identity.issuerBackendUrl=${camunda.tasklist.identity.issuerBackendUrl:$
 
 #---
 spring.config.activate.on-profile=dev
```

Comment:
I would rather to keep it unless you have a strong opinion to remove it (in case we have to add some property only for dev, is quite clear where to add it without breaking the file), wdyt?